ftw/referencewidget/tests/builders.py

```diff
@@ -7,3 +8,9 @@ class SampleContentBuilder(DexterityBuilder):
 
 registry.builder_registry.register(
     'refwidget sample content', SampleContentBuilder)
+
+
+class NewsItemBuilder(ArchetypesBuilder):
```

Comment:
@Nachtalb Well... I now this is petty... can you fix the naming. Everything else 👍

Reply:
Sry for that, done 👍